.github/workflows/code_style.yml

```diff
@@ -26,13 +26,6 @@ jobs:
         run: |
           java -jar google-java-format-1.9-all-deps.jar --set-exit-if-changed -a -i $(find . -type f -name "*.java")
 
-      - name: Create code style diff
```

Comment:
Maybe it would be better to use the [googlejavaformat-action](https://github.com/axel-op/googlejavaformat-action#readme)

Reply:
great suggestions, fixed